helpers/github.ts

```diff
@@ -394,6 +399,28 @@ export async function createDevPoolIssue(projectIssue: GitHubIssue, projectUrl:
     });
     console.log(`Created: ${createdIssue.data.html_url} (${projectIssue.html_url})`);
 
+    // Fetch the list of organizations the user belongs to
+     const userOrgs = await octokit.rest.orgs.listForUser({
+     username: createdIssue.data.user?.login as string,
+    });
```

Comment:
You need to test code before requesting reviews. I'm pretty certain this doesn't work for GitHub Apps.

Reply:
I used jest test to ensure the functionality is working. the test passed and I assumed it worked fine. Apologies for the inconvenience